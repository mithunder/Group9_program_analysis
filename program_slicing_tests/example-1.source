module example_1 :
  read x;
  read y;
  x := 0;
@programslicing="true"
  z := x / y;
  write z
end
