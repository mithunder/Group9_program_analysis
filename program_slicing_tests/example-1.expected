module example_1 :
  read y;
  x := 0;
@programslicing="true"
  z := x / y
end
